ElasticFrameProtocol.h

```diff
@@ -7,10 +7,10 @@
 //  | |____ | || (_| |\__ \| |_ | || (__ | |   | |  | (_| || | | | | ||  __/
 //  |______||_| \__,_||___/ \__||_| \___||_|   |_|   \__,_||_| |_| |_| \___|
 //                                                                  Protocol
-// UnitX @ Edgeware AB 2021
+// UnitX @ Edgeware AB 2021 => agilecontent 2022
```

Comment:
```suggestion
// UnitX @ Edgeware AB 2021 => AgileContent 2022
```

Reply:
It should be Agile Content with a space. I'll change.